README-en.md

```diff
@@ -0,0 +1,382 @@
+
+### Motherboard
+ * Labels of **P2** and **P6** on the board are swapped. The correct labels should be, **P2** is I2C (5V) and **P6** is I2C (3.3V).
+ * Labels of **P7** and **P8** on the board are swapped. The correct labels should be, **P7** is Grove-I/O and **P8** is Grove-UART.
+ * If you are using batteries as your power supply, make sure the polarity of the batteries are properly placed. Reverse voltages may cause permanent damages to the boards.
+ * If the board is powered externally at 3.3V, make sure to not draw power more than 600mA (total power draw, including the CPU core's consumption) to avoid any damage to the power regulator.
+ * Ensure the polarity of the power source's connections as well as the voltage before making connections to the board.
+ * Some parts may missing from the board, you may add them as needed.
+
+---
+
+## Hardware Specifications
+### CPU core
+[cpuboard] (https://user-images.githubusercontent.com/135050/58088657-1bcf7080-7bfe-11e9-89f4-5e6bbde1a363.jpg)
+
+The [Raytac MDBT42Q] (http://www.raytac.com/product/ins.php?index_id=31) module is the main processor of this development board.
+Adafruit's nRF52 Arduino library is compatible with this module and can be used for development through Arduino IDE. It also has JTAG pins available to be connected via J-LINK etc.
+You can also develop everything from scratch using Nordic's nRF5 SDK. Both configurations of the board are supported; CPU core as a standalone as well as paired with the daughter board as a complete development board of LINE Things.
+
+#### Pin assignment
+[cpuboard_pinassign] (https://user-images.githubusercontent.com/135050/58088655-1a9e4380-7bfe-11e9-90d4-530f84e56242.png)
+
+#### Technical Specification
+The following is CPU core's specification.
+
+| Item | Value |
+---- | ----
+Operational voltage | 1.7 to 3.6 V |
+| Maximum RF Power | 4dBm |
+| Flash size | 512KB |
+| RAM size | 64KB |
+
+### Daughter board
+[daughterboard] (https://user-images.githubusercontent.com/135050/58088621-078b7380-7bfe-11e9-8d0d-8d9c216770b4.jpg)
+
+Mounted on the daughter board are LEDs, tactile switches, temperature sensors, and an OLED screen.
+**Please ensure the orientation of the CPU core when using the daughter board. Permanent damages may occur if placed incorrectly.**
+
+#### Technical Specification
+| Item | Value |
+---- | ----
+Power source | Battery / USB / external power supply |
+Operational voltage (battery) | 3V (2 single AAA batteries 1.5V in series) |
+Operational voltage (USB) | 5V |
+Operational voltage (external supply) | 5V |
+| Maximum continuous current consumption (battery powered / motor not used) | 0.6A |
+| Maximum continuous current consumption (when using motor) | 2.6A |
+
+## Peripherals on the daughter board
+**NOTE 5V systems and 3.3V systems cannnot be directly connected to one another.**
```

Comment:
There are 3.3V and 5V devices on the board.
On board battery can only supply 3.3V. So, can not work 5 V device.for example, motors.
When use external or USB power, 5V and 3.3V are provided and all devices can be used.